### PR TITLE
Issues/kp 347 merge new search

### PR DESCRIPTION
**Pull request by amirhossein-haerian**
This PR contains all of the changes for the new search,

There are some specific changes needed to have the beta version of the new search alongside the old search based on this sketch,
https://www.figma.com/design/1zDayarQfV6TkAIMr2JRCC/Om-kursen-sidorna?node-id=3250-6094&node-type=CANVAS&t=ggcwyLWPO21faE6F-0

If you want to see the latest changes I recommend to see the last commit changes.

### CODE REVIEW

File: i18n/messages.en.js [hunk 1]
```diff
@@ -315,12 +335,12 @@ const messages = {
...
     errorEmpty: {
       header: 'Your search returned no results',
-      help: 'For help, see the link below: Instructions for searching',
+      help: '',
     },
     errorOverflow: {
       header: 'There were too many results',
       text: 'There are too many courses that match your search critera. Please specify more characters/digits in the course name or course code (example of course code: SF1624).',
-      help: 'For help, see the link below: Instructions for searching',
+      help: '', // we don't have this link anymore so we should decide what we are going to show as a help text
```
Comment (karlandindrakryggen) [inline i18n/messages.en.js:343]: Look into if we still use the help test. If not; remove it from messages.*.js files.

File: config/commonSettings.js [hunk 2]
```diff
@@ -16,13 +16,17 @@ module.exports = {
   proxyPrefixPath: {
     uri: studentRoot,
     courseSearch: `${studentRoot}/sokkurs`,
+    newSearchPage: `${studentRoot}/sokkurs-ny-design`,
```
Comment (karlandindrakryggen) [inline config/commonSettings.js:19]: Change to  `/sokkurs-beta` in all places where we have `/sokkurs-ny-design` right now.

**Review by axelbjo — APPROVED** [review]

File: config/commonSettings.js (hunk 2 above)
Comment (amirhossein-haerian) [reply config/commonSettings.js]: It is done now, that was a good Idea ⭐️

File: i18n/messages.en.js (hunk 1 above)
Comment (amirhossein-haerian) [reply i18n/messages.en.js]: I checked the code, in the old version we have a collapsable option for showing search instructions but in the new version of the search we do not have it (I mean in the result page where we show the error messages) so the temporary solution is to have two versions of the messages property for old and beta version of the search.
I implemented the required changes and I also added a comment to remove it in the future.

**Review by karlandindrakryggen — APPROVED** [review]

**Pull request merged**